### DISCUSSION TITLE
Trusted domain error - where is the config.php?

### DISCUSSION THREAD

**Discussion by wisdomlight** [nextcloud/all-in-one#4127] [Questions] [answered]
**Edit:
Resolved itself after updating the noip client.**

---
When I go to my domain-name it takes me to the apache2 page
When I go to my domain-name/nextcloud it resolves itself to  https://domain-name/nextcloud/index.php and I am getting this 
```
Access through untrusted domain

Please contact your administrator. If you are an administrator, edit the "trusted_domains" setting in config/config.php like the example in config.sample.php.

Further information how to configure this can be found in the [documentation](https://docs.nextcloud.com/server/27/go.php?to=admin-trusted-domains).
```
Is [this](https://github.com/nextcloud/all-in-one#how-to-change-the-domain) the correct documentation to fix the possible problem?
in nextcloud-aio-apache /var/www/html/config/config.php - my domain is in the overwritehost and the overwriteprotocol is https
the domain is also in the trusted domains array

The file to be edited by this command already has the correct domainn name
```
sudo docker run -it --rm --volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config:rw alpine sh -c "apk add --no-cache nano && nano /mnt/docker-aio-config/data/configuration.json"
```

**Comment by szaimen** [comment, marked as answer]
> Edit:
> Resolved itself after updating the noip client.
